src/quick-lint-js/fe/parse.cpp

```diff
@@ -466,6 +466,60 @@ void Parser::error_on_invalid_as_const(Expression* ast,
   }
 }
 
+void Parser::warn_on_dot_operator_after_optional_chain(
+    Expression::Dot *ast) {
+  Expression *lhs = ast->child_;
+  Source_Code_Span operator_span_ = ast->operator_span_;
```

Comment:
Nit: Parameters and local variables should not have a trailing underscore:
```suggestion
  Source_Code_Span operator_span = ast->operator_span_;
```